File: ReserveService/src/main/resources/domain/Screening.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package domain;

import java.util.Date;
import java.util.List;

/**
 *
 * @author Tom
 * @email
 * @version 0.0.1
 */
public class Screening {

    private long id;
    private String movieTitle;
    private Date startTime;
    private Date endTime;
    private double price;
    private List<Seat> seats;
}

File: movie-app/src/app/models/screening.ts
import {Seat} from "./seat";

export class Screening {
    id: number;
    movieTitle: string;
    startTime: Date;
    endTime: Date;
    price: number;
    seats: Seat[];

    /*
    public toString(): string {
        return this.movieTitle + " - " + this.startTime + " till " + this.endTime;
    }
    */
}
